blosc/frame.c

```diff
@@ -462,7 +463,7 @@ int get_header_info(blosc2_frame *frame, int32_t *header_len, int64_t *frame_len
     }
 
     // Sanity check for compressed sizes
-    if ((*cbytes < 0) || (*nbytes > 0 && *cbytes == 0) || ((int64_t)*nchunks * *chunksize < *nbytes)) {
+    if ((*cbytes < 0) || ((int64_t)*nchunks * *chunksize < *nbytes)) {
```

Comment:
With the support of zero and NaN run-lengths as special values in frame offsets, `(*nbytes > 0 && *cbytes == 0)` is perfectly valid.  I suppose this is the reason why fuzzer tests are complaining.  Possibly the the frames in `tests/fuzz/corpus` needs to be redone because the format of frames has changed. @nmoinvaz ?